lbry/lbry/extras/daemon/comment_client.py

```diff
@@ -41,17 +41,30 @@ def sign_comment(comment: dict, channel: Output):
     pieces = [timestamp, channel.claim_hash, comment['comment'].encode()]
     digest = sha256(b''.join(pieces))
     signature = channel.private_key.sign_digest_deterministic(digest, hashfunc=hashlib.sha256)
-    comment['signature'] = binascii.hexlify(signature).decode()
-    comment['signing_ts'] = timestamp.decode()
+    comment.update({
+        'signature': binascii.hexlify(signature).decode(),
+        'signing_ts': timestamp.decode()
+    })
 
 
-async def jsonrpc_post(url: str, method: str, **params) -> any:
+def sign_abandon_comment(body: dict, channel: Output):
+    pieces = [body['comment_id'].encode(), channel.claim_hash]
+    digest = sha256(b''.join(pieces))
+    signature = channel.private_key.sign_digest_deterministic(digest, hashfunc=hashlib.sha256)
+    body.update({
+        'signature': binascii.hexlify(signature).decode()
+    })
```

Comment:
Is it possible to refactor this so that creating a comment and abandoning a comment use the same signing function? It seems like it should be possible to make this API symmetrical.

Reply:
done